components/BackButton.tsx

```diff
@@ -1,14 +1,15 @@
 import { FC } from 'react';
 import { LeftChevron } from './icons';
-import { useRouter } from 'next/router';
 
-const BackButton: FC = () => {
-  const router = useRouter();
+interface Props {
+  callback: () => void;
```

Comment:
Rename for consistency with actual `button`. Make other occurences match.
```suggestion
  onClick: () => void;
```